ui/src/views/plugins/quota/EditTariffValueWizard.vue

```diff
@@ -112,7 +112,7 @@ export default {
 
         this.loading = true
 
-        api('quotaTariffUpdate', {}, 'POST', params).then(json => {
+        api('quotaTariffUpdate', params, 'GET').then(json => {
```

Comment:
Isn't Post desirable?

Reply:
The same idea, the POST method is not supported here?

Reply:
@rhtyd @utchoang 

Done. I tested with POST, and works fine. Thanks for the suggestion.